db/migrate/20230901195553_add_encrypted_user_attributes_to_oauth_sessions.rb

```diff
@@ -0,0 +1,5 @@
+class AddEncryptedUserAttributesToOAuthSessions < ActiveRecord::Migration[6.1]
+  def change
+    add_column :oauth_sessions, :user_attributes_ciphertext, :text
```

Comment:
I think I would prob add the `encrypted_kms_key` field as well, actually

Reply:
I think it makes it so that data isn't lost when a key is rotated: https://github.com/ankane/kms_encrypted#getting-started